solutions/USACO feb18-teleportation.mdx

```diff
@@ -0,0 +1,49 @@
+---
+id: usaco-807
+source: USACO Bronze 2018 February
+title: Teleportation
+author: Ananth Kashyap
+---
+
+[Official Analysis](http://www.usaco.org/current/data/sol_teleport_bronze_feb18.html)
+
+## Implementation
+
+**Time Complexity:** $\mathcal{O}(1)$
+
+<LanguageSection>
+
+<PySection>
+
+```python
+# Take in data using Python file i/o system
+file_in = open('teleport.in')
+data = file_in.read().strip().split('\n')
+a, b, x, y = map(int, data[0].split(' '))
+
+# Manually sort a, b to be increasing order
+if a > b:
+	a, b = b, a
+
+# Manually sort x, y to be increasing order
+if x > y:
+	x, y = y, x
+
+# Set base distance as distance needed to travel without using teleporter
```

Comment:
```suggestion
# The base distance is abs (a - b)
```

Reply:
same as above, this comment isn't that helpful

Reply:
Yea, fine as is as well.